timemachine/lib/potentials.py

```diff
@@ -34,6 +36,27 @@ def bound_impl(self, precision):
         return custom_ops.BoundPotential(self.unbound_impl(precision), self.params)
 
 
+class SummedPotential:
+    def __init__(self, potentials: List[CustomOpWrapper], params_init: List[NDArray]):
+        self._potentials = potentials
+        self._sizes = [ps.size for ps in params_init]
+        self.params = None
+
+    def bind(self, params: List[NDArray]) -> "SummedPotential":
```

Comment:
Should return self based on the type hint

Reply:
Ah, good catch. Fixed in 4d7e4a8.